src/dmd/tokens.d

```diff
@@ -24,539 +24,292 @@ import dmd.utf;
+    uns8,
+    int16,
+    uns16,
+    int32,
+    uns32,
+    int64,
+    uns64,
+    int128,
+    uns128,
+    float32,
+    float64,
+    float80,
+    imaginary32,
+    imaginary64,
+    imaginary80,
+    complex32,
+    complex64,
+    complex80,
+    char_,
+    wchar_,
+    dchar_,
+    bool_,
 
     // 160
     // Aggregates
-    TOKstruct,
-    TOKclass,
-    TOKinterface,
-    TOKunion,
-    TOKenum,
-    TOKimport,
-    TOKalias,
-    TOKoverride,
-    TOKdelegate,
-    TOKfunction,
-    TOKmixin,
-    TOKalign,
-    TOKextern,
-    TOKprivate,
-    TOKprotected,
-    TOKpublic,
-    TOKexport,
-    TOKstatic,
-    TOKfinal,
-    TOKconst,
-    TOKabstract,
-    TOKdebug,
-    TOKdeprecated,
-    TOKin,
-    TOKout,
-    TOKinout,
-    TOKlazy,
-    TOKauto,
-    TOKpackage,
-    TOKmanifest,
-    TOKimmutable,
+    struct_,
+    class_,
+    interface_,
+    union_,
+    enum_,
+    import_,
+    alias_,
+    override_,
+    delegate_,
+    function_,
+    mixin_,
+    align_,
+    extern_,
+    private_,
+    protected_,
+    public_,
+    export_,
+    static_,
+    final_,
+    const_,
+    abstract_,
+    debug_,
+    deprecated_,
+    in_,
+    out_,
+    inout_,
+    lazy_,
+    auto_,
+    package_,
+    manifest,
+    immutable_,
 
     // 191
     // Statements
-    TOKif,
-    TOKelse,
-    TOKwhile,
-    TOKfor,
-    TOKdo,
-    TOKswitch,
-    TOKcase,
-    TOKdefault,
-    TOKbreak,
-    TOKcontinue,
-    TOKwith,
-    TOKsynchronized,
-    TOKreturn,
-    TOKgoto,
-    TOKtry,
-    TOKcatch,
-    TOKfinally,
-    TOKasm,
-    TOKforeach,
-    TOKforeach_reverse,
-    TOKscope,
-    TOKon_scope_exit,
-    TOKon_scope_failure,
-    TOKon_scope_success,
+    if_,
+    else_,
+    while_,
+    for_,
+    do_,
+    switch_,
+    case_,
+    default_,
+    break_,
+    continue_,
+    with_,
+    synchronized_,
+    return_,
+    goto_,
+    try_,
+    catch_,
+    finally_,
+    asm_,
+    foreach_,
+    foreach_reverse_,
+    scope_,
+    onScopeExit,
+    onScopeFailure,
+    onScopeSuccess,
 
     // 215
     // Contracts
-    TOKinvariant,
+    invariant_,
 
     // Testing
-    TOKunittest,
+    unittest_,
 
     // Added after 1.0
-    TOKargTypes,
-    TOKref,
-    TOKmacro,
+    argTypes,
+    ref_,
+    macro_,
 
     // 221
-    TOKparameters,
-    TOKtraits,
-    TOKoverloadset,
-    TOKpure,
-    TOKnothrow,
-    TOKgshared,
-    TOKline,
-    TOKfile,
-    TOKfilefullpath,
-    TOKmodulestring,
-    TOKfuncstring,
-    TOKprettyfunc,
-    TOKshared,
-    TOKat,
-    TOKpow,
-    TOKpowass,
-    TOKgoesto,
-    TOKvector,
-    TOKpound,
+    parameters,
+    traits,
+    overloadSet,
+    pure_,
+    nothrow_,
+    gshared,
+    line,
+    file,
+    fileFullPath,
+    moduleString,
+    funcString,
+    prettyFunc,
+    shared_,
+    at,
+    pow,
+    powass,
+    goesto,
+    vector,
+    pound,
 
     // 239
-    TOKinterval,
-    TOKvoidexp,
-    TOKcantexp,
+    interval,
+    voidexp,
+    cantexp,
```

Comment:
@JinShil I wasn't sure here. `voidExp`? `cantExp`?

Reply:
I suggest `voidExp` and `cantExp`.